gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -1398,7 +1398,8 @@ private static void checkContentType(BlobInfo blobInfo) throws IllegalArgumentEx
   byte[] readAllBytes(BlobId blob, BlobSourceOption... options);
 
   /**
-   * Send a batch request.
+   * Send a batch request. If the {@code batchRequest} parameter contains more than 100 deletes the
```

Comment:
I don't think we should document that, nor force it on other `StorageRpc` providers.
I see this change as a necessary internal implementation to avoid an existing known issue (which is likely to be
fixed in the future). Do you think otherwise?